### PR TITLE
@mzikherman => [Loyalty] adds responsiveness (mobile support) to inquiries page

### PR DESCRIPTION
**Pull request by l2succes**
Fixes https://github.com/artsy/reaction-force/issues/100 & https://github.com/artsy/reaction-force/issues/93

### CODE REVIEW

**Comment by xtina-starr** [comment]
mergin'

**Comment by mzikherman** [comment]
Nice, gonna put on my serious CSS/responsive hat and give this a read 🍵

File: src/components/helpers.tsx [hunk 1]
```diff
@@ -14,3 +15,17 @@ export const block = (margin: number = 0) => {
...
+  // using px in breakpoints to maintain uniform units with flexbox-grid
+  // https://zellwk.com/blog/media-query-units/
+  const emSize = sizes[label]
+  accumulator[label] = (strings, ...args) => css`
+    @media (max-width: ${emSize}px) {
+      ${css(strings, ...args)}
+    }
+  `
+  return accumulator
+}, {})
```
Comment (alloy) [inline src/components/helpers.tsx]: Ooh, template strings, funky!

File: src/assets/theme.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      lg: 76,  // rem
+    },
+    breakpoints: {
+      xs: 0,
+      sm: 768,
+      md: 1024,
+      lg: 1200,
+    },
+  },
+}
```
Comment (alloy) [inline src/assets/theme.ts]: Abbreviations are a bit of a pet peeve of mine as they introduce ambiguity to new readers for no real good reason. Can you change these to long form?

**Pull request merged**